src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTestBase.cs

```diff
@@ -39,7 +36,13 @@ public HttpClientHandlerTestBase(ITestOutputHelper output)
         protected HttpClient CreateHttpClient(HttpMessageHandler handler)
         {
             var client = new HttpClient(handler);
-            SetDefaultRequestVersion(client, VersionFromUseHttp2);
+
+            // Always set the default request version to HTTP/2.
+            // The actual version used will be determined by the server (either loopback server or remote server).
+            // Note that if you create the HttpRequestMessage explicitly, you will need to set its Version explicitly
+            // because it defaults to 1.1.
+
+            SetDefaultRequestVersion(client, HttpVersion.Version20);
```

Comment:
`HttpClient.DefaultRequestVersion` is a public API now.  So, this method could be removed and this line replaced with 

```c#
client.DefaultRequestVersion = HttpVersion.Version20;
```